src/registrar/views/application.py

```diff
@@ -365,6 +365,13 @@ class AuthorizingOfficial(ApplicationWizard):
     template_name = "application_authorizing_official.html"
     forms = [forms.AuthorizingOfficialForm]
 
+    def get_context_data(self):
+        context = super().get_context_data()
+        context["is_federal"] = self.application.is_federal()
```

Comment:
**thought (non-blocking)**: This is the second form  that needed `context["is_federal"]`. Maybe that is common enough that we could move it into the base class and provide that context in all of the forms? The change doesn't need to be made here, just noticing that we reached the second gate in the "Rule of Three".

Reply:
Ah good point, and we will likely reach the 3rd gate when doing the .gov domain examples. I can change it as part of that work.